### ISSUE TITLE
Change WSL2 shorcuts

### ISSUE THREAD

**Issue by leolbyy** [microsoft/WSL#10227]
I am on both Mac (working SSH machine) and Windows (causal), and both need connect to linux environment. On Mac, copy and interrupt are handled perfectly well with cmd + c and ctrl +c.

Is it possible to allow user to change WSL shortcuts? For example I can change to alt + c as copy and ctrl + c as interrupt

I am currently running Ubuntu 18.04 on Windows Terminal v 1.18 preview release.
<img width="630" alt="image" src="https://github.com/microsoft/WSL/assets/80955869/8319d68f-657e-47ac-8b1f-dda7dfd85c95">

**Comment by Paultergeist** [comment]
I don't think it has anything to do with WSL, rather it's your terminal emulator. If you are using Terminal app on Windows, you can change shortcuts for copy and paste as explained [here](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions) and [here](https://stackoverflow.com/a/62956033).

**Comment by benhillis** [comment]
Closing — shortcut/keybinding remapping is handled by your terminal emulator, not WSL itself. For Windows Terminal, see https://learn.microsoft.com/windows/terminal/customize-settings/actions and please file requests at https://github.com/microsoft/terminal.
